src/Microsoft.DotNet.Configurer/NuGetCacheSentinel.cs

```diff
@@ -23,7 +23,7 @@ private string NuGetCachePath
             {
                 if (string.IsNullOrEmpty(_nugetCachePath))
                 {
-                    _nugetCachePath = PackageDependencyProvider.ResolvePackagesPath(null, null);
+                    _nugetCachePath = NuGetPathContext.Create(new NullSettings()).UserPackageFolder;
```

Comment:
Actually, jus thinking about it. The global.json scenario is not going to work, even if we understand it here. This means the sentinel will be in the right folder, but because we do a dotnet new, dotnet restore in a temp folder, that restore will not use the nuget cache from global.json.

@eerhardt I think you are fine with your change as is. Like you said, it makes things better in the code and is not breaking anything new. I will open an issue for global.json and offline.